docs/sources/static/configuration/agent-management.md

```diff
@@ -68,6 +68,9 @@ agent_management:
     # Whether to use labels from the label management service. If enabled, labels from the API supersede the ones configured in the agent.
     label_management_enabled: <bool> | default = false
 
+    # Whether to accept HTTP Not Modified (304) responses from the API server. If enabled, the agent will use the cached configuration if the API server responds with a 304. This can be disabled for debugging or testing, but will save network bandwidth and CPU time if enabled.
+    accept_http_not_modified: <bool> | default = false
```

Comment:
any reason not to make this default to `true`?

Reply:
I was going to ask the same. A default to `false` tells me that if a 304 is received, requested resources are sent again instead of redirecting to the cached resources. That seems backwards to me.